docs/fermi_hubbard/experiment_example.ipynb

```diff
@@ -11,7 +11,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
```

Comment:
why is this change included?

Reply:
Ah, leftover from intentional unformatting. Fixed now.